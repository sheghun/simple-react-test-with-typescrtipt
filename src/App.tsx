import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
            <p>Working</p>
            <p>Name of the boy</p>
      </div>
    );
  }
}

export default App;
