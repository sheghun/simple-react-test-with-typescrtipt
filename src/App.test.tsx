import React from 'react';
import * as enzyme from 'enzyme'
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Finds the text', () => {
    const wrapper = enzyme.shallow(<App />)
    expect(wrapper.find('p').text()).toEqual('Working')
})