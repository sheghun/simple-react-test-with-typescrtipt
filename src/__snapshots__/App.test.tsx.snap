// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders working on the p child 1`] = `
<div
  className="App"
>
  <p>
    Working
  </p>
  <p>
    Name of the boy
  </p>
</div>
`;
